### PR TITLE
docs: Update twilight_http example, add alt docker build cmd with metrics enabled

### PR DESCRIPTION
**Pull request by drklee3**
### Description

Updates the twilight_http example on the README.  Also adds a `docker build` command with metrics enabled. 

### Tested

Example now compiles on `twilight_http v0.3.5`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +35,11 @@ to listen via HTTPS, then don't use HTTP.
 Build the dockerfile and then run it:
 
 ```sh
-$ docker build . -t http-proxy
-$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 http-proxy
+docker build . -t http-proxy
+# Or build with the metrics feature enabled
+docker build . --build-arg FEATURES="expose-metrics" -t http-proxy
+
+docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 http-proxy
```
Comment (Gelbpunkt) [inline README.md]:
```suggestion
$ docker build . -t http-proxy
# Or build with the metrics feature enabled
$ docker build . --build-arg FEATURES="expose-metrics" -t http-proxy

$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 http-proxy
```
I'd keep the $ to indicate that this is a shell command and to keep consistency with the part below.

**Review by Gelbpunkt — CHANGES_REQUESTED** [review]
We should probably remove the docker build part from the README and replace it with a reference to the prebuilt ones (`docker.io/twilightrs/http-proxy`).

**Comment by drklee3** [comment]
> We should probably remove the docker build part from the README and replace it with a reference to the prebuilt ones (`docker.io/twilightrs/http-proxy`).

Ah yeah I was not aware of the prebuilt images, should I just replace the build commands with `docker run .. twilightrs/http-proxy` commands instead?

**Comment by Gelbpunkt** [comment]
> > We should probably remove the docker build part from the README and replace it with a reference to the prebuilt ones (`docker.io/twilightrs/http-proxy`).
> 
> Ah yeah I was not aware of the prebuilt images, should I just replace the build commands with `docker run .. twilightrs/http-proxy` commands instead?

Yes, that makes more sense. If you do so make sure to mention the different tags for metrics, ARM, API v8 and the old v6.

**Review by Gelbpunkt — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -35,16 +32,21 @@ to listen via HTTPS, then don't use HTTP.
...
+# Or with metrics enabled
+$ docker run -itd -e DISCORD_TOKEN="my token" -p 3000:80 twilightrs/http-proxy:metrics
 ```
 
 This will set the discord token to `"my token"` and map the bound port to port
 3000 on the host machine.
 
+Images come in multiple different variants for metrics, ARM, Discord API v8 and
+v6. You can use these with their corresponding image tags found on
+[Docker Hub][docker-hub-tags].
```
Comment (7596ff) [inline README.md]: Could you either rename this link to show that it goes to the tags page specially, or use the same link as Docker Hub above?

**Review by 7596ff — CHANGES_REQUESTED** [review]
Thanks for the PR, one small thing:

**Review by 7596ff — APPROVED** [review]

**Pull request merged**